ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -234,6 +234,52 @@
    */
   CompletableFuture<ServerInfo> serverInfo();
 
+  /**
+   * Creates a connector.
+   *
+   * <p>If a non-200 response is received from the server, the {@code CompletableFuture} will be
+   * failed.
+   *
+   * @param connectorName name of the connector
+   * @param isSource true if the connector is a source connector, false if it is a sink connector
+   * @param properties connector properties
+   * @return result of connector creation
+   */
+  CompletableFuture<Void> createConnector(
+      String connectorName, boolean isSource, Map<String, String> properties);
```

Comment:
I believe we use `Map<String, Object>` to pass properties elsewhere in the client interface. Is there a reason for using `<String, String>` here, or should we keep this consistent with the other interface methods?

Reply:
Good catch! Will update the PR.